src/main/java/org/candlepin/subscriptions/tally/billing/BillableUsageController.java

```diff
@@ -90,16 +96,36 @@ public BillableUsageRemittanceEntity getLatestRemittance(BillableUsage billableU
   }
 
   private BillableUsageCalculation calculateBillableUsage(
-      double measuredTotal, double currentRemittedValue) {
-    double adjustedMeasuredTotal = Math.ceil(measuredTotal);
-    double billableValue = adjustedMeasuredTotal - currentRemittedValue;
-    if (billableValue < 0) {
-      // Message could have been received out of order via another process,
-      // or on re-tally we have already billed for this usage and using
-      // credit. There's nothing to bill in this case.
-      billableValue = 0.0;
+      double measuredTotal, BillableUsage usage, BillableUsageRemittanceEntity remittance) {
+    var tagMetricOptional =
+        tagProfile.getTagMetric(
+            usage.getProductId(), Measurement.Uom.fromValue(usage.getUom().value()));
+    double tagFactor =
+        tagMetricOptional
+            .map(TagMetric::getBillingFactor)
+            .orElse(1.0); // get configured billingFactor in tag_profile yaml
+    double billableValue;
+    double remittedValue;
+    var currentRemittedValue = remittance.getRemittedValue();
+
+    if (tagFactor != 1.0) {
+      var prevBilled = currentRemittedValue / remittance.getBillingFactor(); // previously billed
```

Comment:
What is the `remittance.getBillingFactor()` going to be here?  Will it be the same factor as what's in the tag profile or the reciprocal?

I.e.: is this line doing something inadvertent like `currentRemittedValue / 0.25` which would actually multiply it by four.

Reply:
Okay, thanks.